pkg/repository/local_git.go

```diff
@@ -24,6 +29,23 @@ type LocalExec struct{}
 
 func (*LocalExec) RunCommand(ctx context.Context, dir string, name string, arg ...string) ([]byte, error) {
 	c := exec.CommandContext(ctx, name, arg...)
+	c.Cancel = func() error {
```

Comment:
I think CircleCI is complaining about the existence of `Cancel`

Reply:
yes, i am investigating